### PR TITLE
fix: automodOnSpecificServer

### LINKED ISSUE #107: Automod is not server specific
Currently automod will trigger in all server, not just the wanted one.

### PR DESCRIPTION
**Pull request by KommentatorForAll**
fixes #107

### CODE REVIEW

File: src/modules/automod/message.js [hunk 1]
```diff
@@ -116,7 +117,7 @@ async function handlePings(client, message) {
 export async function messageCreate(client, message) {
     if (message.author.bot) return;
     // If this is a DM we can ignore these
```
Comment (WilsontheWolf) [inline src/modules/automod/message.js:119]: Maybe update the comment to reflect what is done.

Comment (KommentatorForAll) [inline src/modules/automod/message.js]:
```suggestion
    // If this is a DM or not on the main server, we can ignore these

Comment (WilsontheWolf) [reply src/modules/automod/message.js]: Why are you reviewing your own pr?

File: src/modules/automod/message.js [hunk 2]
```diff
@@ -115,8 +116,8 @@ async function handlePings(client, message) {
  */
 export async function messageCreate(client, message) {
     if (message.author.bot) return;
-    // If this is a DM we can ignore these
-    if (!message.guild) return;
+    // If the message is outside of the default guild we can ignore it
+    if (!message.guild || message.guild.id !== config.defaultGuild) return;
```
Comment (WilsontheWolf) [inline src/modules/automod/message.js:120]:
```suggestion
    if (message.guild?.id !== config.defaultGuild) return;
```
Shorter code

**Review by WilsontheWolf — APPROVED** [review]

**Pull request merged**